### DISCUSSION TITLE
Get `forward` like in the game state integration

### DISCUSSION THREAD

**Discussion by lucasreppewelander** [saul/demofile#368] [Questions]
Hello, I'm trying to parse where a player is looking and from what I can see I can only get the `pitch` and `yaw. At first glance, thinking from a 2d perspective, this would be the best way to determine where the player is looking, but I seem to fail to get the correct value (i.e the player on my canvas is looking at the wrong direction).

I do have a function which works with the CSGO game state integration and there I get the `forward` in this form:
```
position: '-258.71, -2152.25, -173.31',
forward: '0.41, 0.91, -0.02',
```
I've seen that there is something like _**aimDirection**_ on the `usercmd` event or similar but I fail to understand how to use this on a certain tick/event? And I'm not sure if this is even what I am looking for.

Do you guys have any idea of how to obtain the forward from either the pitch & yaw from `player.eyeAngles` or from another prop on the player? I'm trying to get this player information on certain events like `player_death` and etc.

Thanks

**Comment by saul** [comment]
`eyeAngles` should give you the direction that the player is looking in degrees. If you're working in 2D, then you only need to look at the yaw. Have you tried using `cl_showpos 2` to check what direction 0º/90º/etc yaw is?
